scout/demo/643594.config.yaml

```diff
@@ -47,6 +49,8 @@ peddy_ped: scout/demo/643594.peddy.ped
 peddy_ped_check: scout/demo/643594.ped_check.csv
 peddy_sex_check: scout/demo/643594.sex_check.csv
 
+delivery_report: scout/demo/delivery_report.html
```

Comment:
This path should be a full path, how can we solve that?

Reply:
Hi, when parsing the yaml file you can do something like
```python
import os

full_path = os.path.abspath(config['delivery_report'])
```

Reply:
Do you think we should change the implementation in order to satisfy the demo-case-config delivery report? I guess we could skip it also